include/pka_config.h

```diff
@@ -178,8 +178,23 @@
 // TRNG Control Register Value; Set bit 10 to start the EIP-76 a.k.a TRNG
 // engine, gathering entropy from the FROs.
 #define PKA_TRNG_CONTROL_REG_VAL            0x00000400
+
+/* TRNG Control bit */

```

Comment:
Use "// comment" instead of "/* comment */" to be consistent with the rest of the code

Reply:
Ah, yes, good catch!